lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -11,10 +17,31 @@
     </div>
 </div>
 
-<div class="form-instructions create-team-instructions">
-    <p class="copy">
-        <%- gettext("Enter information to describe your team. You cannot change these details after you create the team.") %></p>
-</div>
+<% if (isNew) { %>
+    <div class="form-instructions create-team-instructions">
+        <p class="copy">
+            <%- gettext("Enter information to describe your team. You cannot change these details after you create the team.") %></p>
+    </div>
+<% } else { %>
+    <div class="team-info wrapper-msg-fixed is-incontext urgency-low warning" tabindex="-1">
+        <div class="msg">
+            <div class="msg-content">
+                <div class="copy">
+                <p>
+                    <%=
+                        interpolate_text(
+                            _.escape(gettext("The team that you are editing has {total_members} members. Communicate important changes to students prior to updating the team when possible.")),
```

Comment:
@muzaffaryousaf Here's my suggestion:
""The team that you are editing has {total_members} members. Notify team members prior to making significant changes."